modules/ipi-preparing-reinstall-cluster-bare-metal.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]
+
+= Preparing to reinstall a cluster on bare-metal
+Before you reinstall a cluster on bare-metal, you must perform cleanup operations.
+
+.Procedure
+. Remove or reformat the disks for the bootstrap, master, and worker nodes. If you are working in a hypervisor environment, you will need to re-add any disks you removed. You are now starting with clean disks for the bootstrap, master, and worker nodes.
+. Delete the artifacts that the previous installation generated:
+[source,terminal]
+----
+$ cd ; /bin/rm -rf auth/ bootstrap.ign master.ign worker.ign metadata.json .openshift_install.log .openshift_install_state.json
+----
+[start=3]
```

Comment:
By using the  `+` you should be able to delete `[start=3]` here.

Reply:
Done.